operators/src/upgrade/common/constants.rs

```diff
@@ -44,3 +46,29 @@ pub(crate) const UPGRADE_OPERATOR_SERVICE: &str = "upgrade-operator-service";
 pub(crate) const UPGRADE_OPERATOR_SERVICE_PORT: i32 = 8080;
 /// Service internal port constant for upgrade operator.
 pub(crate) const UPGRADE_OPERATOR_INTERNAL_PORT: i32 = 8080;
+
+pub(crate) const NODE_LABEL: &str = "openebs.io/engine=mayastor";
+
+pub(crate) const DEFAULT_VALUES_PATH: &str = "./";
+
+pub(crate) fn components() -> HashMap<String, Vec<String>> {
+    let core_components: Vec<String> = vec![
+        "core_agent".to_string(),
```

Comment:
nit: you can remove all this to_string here and simply convert at the end, eg:
```rust
let core_components = vec!["core_agent", "rest_api"]; ...
components.insert("core_components".to_string(), core_components.into_iter().map(ToString::to_string).collect())
```
I don't know how these are used but I wonder if the best thing really is to make use of enums rather than generic strings.